test/extended/networking.sh

```diff
@@ -20,6 +20,8 @@ source "${OS_ROOT}/hack/util.sh"
 source "${OS_ROOT}/hack/common.sh"
 os::log::install_errexit
 
+USE_DEBUGGER=${OPENSHIFT_USE_DEBUGGER:-false}
```

Comment:
why have two named envvars? why not just use `OPENSHIFT_USE_DEBUGGER` directly? Also, if we're inventing envvar names, I'd prefer a more targeted one (`NETWORK_DEBUG=...` or something)